worker/caasoperatorprovisioner/worker.go

```diff
@@ -186,7 +194,11 @@ func (p *provisioner) ensureOperators(apps []string) error {
 			appPasswords = append(appPasswords, apicaasprovisioner.ApplicationPassword{Name: app, Password: password})
 		}
 
-		config, err := p.makeOperatorConfig(app, password)
+		var prevCfg caas.OperatorConfig
```

Comment:
as discussed, let's revert all this stuff

Reply:
This part is still needed.